src/structures/cl_args.cpp

```diff
@@ -17,13 +17,7 @@ namespace io {
 const unsigned CLArgs::max_exploration_level = 5;
 
 // Default values.
-CLArgs::CLArgs()
-  : check(false),
-    geometry(false),
-    router(ROUTER::OSRM),
-    nb_threads(4),
-    exploration_level(5) {
-}
+CLArgs::CLArgs(){}
```

Comment:
We can probably remove this altogether from both the cpp and header file and let the compiler generate a default ctor.